packages/vscode-extension/src/commands/Deploy.ts

```diff
@@ -8,6 +8,8 @@ import { actionResultStatus, ApplicationConstants, ProjectInfoServive, CLIConfig
 import BaseAction from './BaseAction';
 import { EOL } from 'os';
 
+const COMMAND_NAME = 'deploy';
+
 const DEPLOY_COMMAND = {
 	NAME: 'project:deploy',
 	OPTIONS: {
```

Comment:
Maybe we can already change the value of FLAGS.APPLY_INSTALLATION_PREFS since cli already supports applyinstallprefs flag

Reply:
Completely agree!